compiler/ast/src/main/kotlin/com/uber/xprocessing/ext/XOverrides.kt

```diff
@@ -105,8 +105,7 @@ object XOverrides {
         // one that declares the same method, and check that we haven't reached mA. We compare
         // the enclosing elements rather than the methods themselves for the reason described
         // at the start of the method.
-        var inherited: XMethodElement? = null
-        (inherited != null && overridden.enclosingElement != inherited.enclosingElement)
+        false
```

Comment:
please be sure to test a local build of this in monorepo before we land this

Reply:
Confirmed that all three of uber's big apps (rider / uber eats / driver) can be built with a motif version running the commit at [5a79929](https://github.com/uber/motif/pull/260/commits/5a79929743cee5e1921531d32692e904e4d489e8)